src/index.spec.ts

```diff
@@ -4,9 +4,149 @@ import {
   removeDuplicatesFromArray,
   createRandomProduct,
   getStarWarsPlanets,
-  createProduct 
+  createProduct,
 } from './index';
 
-test('Testing time!', () => {
+/*test('Testing time!', () => {
   console.log('welcome');
+});*/
+
+describe('isInteger()',() => {
+  it("should return true if value is a number", () => {
+    expect(isInteger(1)).toBe(true);
+    expect(isInteger(-1)).toBe(true);
+    expect(isInteger(0)).toBe(true);
+  });
+  it('should return false if value is a string', () => {
+    expect(isInteger('test')).toBe(false);
+    expect(isInteger('111')).toBe(false);
+    expect(isInteger('-123')).toBe(false);
+  });
+});
+
+describe('toLowerCase()', () => {
+  it('should return a default message if not pass a string', () => {
+    const defaultMessage = 'Please provide a string';
+    expect(toLowerCase('')).toBe(defaultMessage);
+    expect(toLowerCase(null)).toBe(defaultMessage);
+    expect(toLowerCase(undefined)).toBe(defaultMessage);
+  });
+  it('should return a string in lower case', () => {
+    expect(toLowerCase('TEST')).toBe('test');
+    expect(toLowerCase('test')).toBe('test');
+    expect(toLowerCase('TeSt')).toBe('test');
+  });
+});
+
+describe('removeDuplicatesFromArray()', () => {
+  it('should throw an error if not array provided', () => {
+    const errorMessage = 'please provide an array of numbers or strings';
+    expect(() => removeDuplicatesFromArray(null)).toThrowError(errorMessage);
+    expect(() => removeDuplicatesFromArray(undefined)).toThrowError(errorMessage);
+    expect(() => removeDuplicatesFromArray({} as any)).toThrowError(errorMessage);
```

Comment:
I think Here you could use jest-in-case as well. 

We usually don't need to invoke function through an arrow function i.e. `() => removeDuplicatesFromArray(null)`, so using something like `expect(removeDuplicatesFromArray(null)).toThrowError(errorMessage);` should be OK.